### PR TITLE
Read meta::pure::functions::date::firstDayOfYear as a Date function

### PR DESCRIPTION
**Pull request by YannanGao-gs**
## Summary

Read meta::pure::functions::date::firstDayOfYear as a Date function

## How did you test this change?

- [x] Test(s) added
- [ ] Manual testing (please provide screenshots/recordings)
- [ ] No testing (please provide an explanation)

1.

https://github.com/finos/legend-studio/assets/73408381/14ac1234-c468-4f3c-8ad4-72a55b80893b

2. fix the css

https://github.com/finos/legend-studio/assets/73408381/23bf0a33-fc22-41c0-acee-897db01062fc

3.

https://github.com/finos/legend-studio/assets/73408381/792b0e2d-c07f-446b-9262-64d0b0cb2790

### CODE REVIEW

File: .changeset/curly-cars-burn.md [hunk 1]
```diff
@@ -0,0 +1,5 @@
+---
+'@finos/legend-query-builder': patch
+---
+
+Read meta::pure::functions::date::firstDayOfYear as a Date function
```
Comment (MauricioUyaguari) [inline .changeset/curly-cars-burn.md:5]: lets update this to include the month function we including

**Review by MauricioUyaguari — CHANGES_REQUESTED** [review]

File: .changeset/curly-cars-burn.md (hunk 1 above)
Comment (MauricioUyaguari) [inline .changeset/curly-cars-burn.md:5]:
````suggestion
Read `meta::pure::functions::date::firstDayOfYear` and `meta::pure::functions::date::firstDayOfMonth` as a Date functions
````

**Comment by MauricioUyaguari** [comment]
@YannanGao-gs  looks good, lets rebase. thanks!

**Comment by YannanGao-gs** [comment]
> @YannanGao-gs looks good, lets rebase. thanks!

rebased thanks!

**Review by MauricioUyaguari — APPROVED** [review]

**Pull request merged**